dev/com.ibm.ws.microprofile.faulttolerance/src/com/ibm/ws/microprofile/faulttolerance/impl/package-info.java

```diff
@@ -11,7 +11,7 @@
 /**
  * @version 2.0
```

Comment:
Shouldn't this version match the one below?

Reply:
Err, yeah...

Reply:
Yes it should.